CalendarApp/GoogleCalendarService/Manager/EventManager.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Google.Apis.Calendar.v3;
+using Google.Apis.Calendar.v3.Data;
+using Google.Apis.Requests;
+using GoogleCalendarService.Extensions;
+
+namespace GoogleCalendarService.Manager
+{
+    public class EventManager
+    {        
+        private readonly CalendarService _service;
+
+        public EventManager(CalendarService service)
+        {
+            _service = service;
+        }
+
+        //Need this for UpdateEventMethod
```

Comment:
If this comment is really important then it should be more descriptive. Otherwise it should be removed